services/harmony/bin/prebuild

```diff
@@ -3,11 +3,27 @@
 # sets up the project to be ready for compiling and building docker image
 #
 
-rimraf built/*
-copyfiles -u 2 -e "../../db/*.sqlite3" "../../db/**/*" built
-# need to copy the db directory into the servcies/harmony directory as well to make knex migrations work because knex is stupid
-copyfiles -u 2 -e "../../db/*.sqlite3" "../../db/**/*" built/services/harmony
-copyfiles -u 2 "../../bin/start-harmony-in-container" built
-copyfiles -u 2 "../../config/services.yml" built
-copyfiles -u 4 -e "../../packages/util/node_modules/**/*" -e "../../packages/util/built/**/*" -e "../../packages/util/test/**/*" -e "../../packages/util/coverage/**/*" "../../packages/util/**/*"  built/packages/util
-copyfiles -e "./test/**/*" -e "./node_modules/**/*" -e "./coverage/**/*" -e "./built/**/*" -e "./bin/**/*" -e "./fixtures/**/*" "./**/*" built/services/harmony
```

Comment:
If we're no longer using copyfiles anywhere I think there might be a dependency we can drop from listing in our documentation.

Reply:
Other core services are still using the copyfiles in their prebuild script. I don't want to do too much unrelated stuff in this PR. We can write a new ticket and do the consolidation separately.